spec/requests/users_controller_spec.rb

```diff
@@ -66,7 +66,8 @@
                 expect(user.name).to be <= next_user.name
               else
                 case user.role
-                when "super_admin" then expect(next_user.role).to eq "editor"
+                when "super_admin" then expect(next_user.role).to eq "organisation_admin"
```

Comment:
Also the array higher up in the class for the Email Domain List should be using the %W format (again, another Rubocop surprise)

Reply:
I think this test is showing that the #index action sorts users it returns in a specific order. It is a bit confusing though...

I can't quite tell which bit you're referring to for the `%W` change, do you have a line number you can refer to?

Reply:
I think rather than doing a clever loop we should write these out as explicit tests... if you have time it would be awesome to have this refactored rather than adding to it.

Reply:
@chao-xian we experimented with this a little but pausing as this PR is blocking Tom's PR for adding the group member management stuff.

Separating them all out into explicit tests is tricky as we lose the test coverage of the sort sequence/categorisation. We also thought about defining some "golden data", i.e. the users list be must exactly this, in this specific order, but that has the same downside of the current approach of not being granular about where failures happen.

Reply:
Ok happy for you to unblock Tom and really appreciate you all trying 👍